src/utils/loadMarket.ts

```diff
@@ -1,14 +1,55 @@
-import { BigInt, Bytes } from '@graphprotocol/graph-ts';
+import {
+  Address, BigInt, Bytes, ethereum,
+} from '@graphprotocol/graph-ts';
 import { Market } from '../../generated/schema';
 
-export default function loadMarket(market: Bytes): Market {
+import { Market as MarketABI } from '../../generated/MarketWETH/Market';
+import { ERC20 } from '../../generated/Auditor/ERC20';
+import { InterestRateModel } from '../../generated/MarketWETH/InterestRateModel';
+
+export default function loadMarket(market: Bytes, event: ethereum.Event): Market {
   const id = market.toHexString();
   let entity = Market.load(id);
   if (entity) return entity;
 
   entity = new Market(id);
   entity.address = market;
-  entity.totalSupply = BigInt.zero();
-  entity.totalFloatingBorrowShares = BigInt.zero();
+  entity.timestamp = event.block.timestamp.toU32();
+  entity.block = event.block.number.toU32();
+
+  const mkt = MarketABI.bind(Address.fromBytes(market));
+  entity.totalSupply = mkt.totalSupply();
+  entity.totalFloatingBorrowShares = mkt.totalFloatingBorrowShares();
+  entity.lastAccumulatorAccrual = mkt.lastAccumulatorAccrual().toU32();
+  entity.floatingAssets = mkt.floatingAssets();
+  entity.floatingDebt = mkt.floatingDebt();
+  entity.earningsAccumulator = mkt.earningsAccumulator();
+  entity.decimals = mkt.decimals();
+  entity.symbol = mkt.symbol();
+  entity.asset = mkt.asset();
+  entity.lastFloatingDebtUpdate = mkt.lastFloatingDebtUpdate().toU32();
+
+  entity.interestRateModel = mkt.interestRateModel();
+  entity.earningsAccumulatorSmoothFactor = mkt.earningsAccumulatorSmoothFactor();
+  entity.treasury = mkt.treasury();
+  entity.treasuryFeeRate = mkt.treasuryFeeRate();
+  entity.maxFuturePools = mkt.maxFuturePools();
+  entity.backupFeeRate = mkt.backupFeeRate();
+  entity.reserveFactor = mkt.reserveFactor();
+  entity.penaltyRate = mkt.penaltyRate();
+  entity.totalFloatingBorrowShares = mkt.totalFloatingBorrowShares();
+
+  const asset = ERC20.bind(mkt.asset());
+  entity.assetSymbol = asset.symbol();
+
+  const irm = InterestRateModel.bind(mkt.interestRateModel());
+  entity.fixedCurveA = irm.fixedCurveA();
+  entity.fixedCurveB = irm.fixedCurveB();
+  entity.fixedMaxUtilization = irm.fixedMaxUtilization();
+  entity.floatingCurveA = irm.floatingCurveA();
+  entity.floatingCurveB = irm.floatingCurveB();
+  entity.floatingMaxUtilization = irm.floatingMaxUtilization();
+  entity.floatingUtilization = new BigInt(2);
```

Comment:
```suggestion
  entity.floatingUtilization = entity.floatingAssets > 0 ? entity.floatingDebt/entity.floatingAssets : 0;
```
move line below `entity.floatingDebt = mkt.floatingDebt();`